spec/features/viewing_timestamps_on_peeps_spec.rb

```diff
@@ -0,0 +1,25 @@
+feature 'Viewing timestamps' do
+  scenario 'A user can see timestamps for peeps' do
+    user = User.create(email: 'test@example.com', password: 'password123', name: 'Test Name', username: 'testuser1')
+    Peep.create(peep_text: 'peeping away 🐥', user_id: user.user_id)
```

Comment:
Could add time.new into your .rb just to make testing easier and then push through the same time into your DB